x-pack/plugins/observability_log_explorer/README.md

```diff
@@ -1,3 +1,68 @@
 # Observability Log Explorer
 
 This plugin provides an app based on the `LogExplorer` component from the `log_explorer` plugin, but adds observability-specific affordances.
+
+## Testing
+
+### Stateful
+
+#### FTR Server
+```
+node scripts/functional_tests_server --config ./x-pack/test/functional/apps/observability_log_explorer/config.ts
+```
+
+#### FTR Runner
+```
+node scripts/functional_test_runner --config ./x-pack/test_serverless/functional/test_suites/observability/config.ts --include ./x-pack/test_serverless/functional/test_suites/observability/observability_log_explorer/index.ts
```

Comment:
**note:** this runner is for the serverless test suite, it should point to the stateful one for both `--config` and `--include` params.